kubernetes-addons/prometheus/values.yaml

```diff
@@ -1,8 +1,8 @@
 serviceAccounts:
```

Comment:
This file is renamed and the content is changed now. Rebase from main should fix this